arches/app/etl_modules/tile_excel_importer.py

```diff
@@ -133,22 +133,29 @@ def process_worksheet(self, worksheet, cursor, node_lookup, nodegroup_lookup):
             [self.loadid],
         )
         return {"name": worksheet.title, "rows": row_count}
+    
+    def validate_uploaded_file(self, file):
+        workbook = load_workbook(filename=default_storage.open(file))
+        graphid = None
+        for worksheet in workbook.worksheets:
+            if worksheet.cell(2, worksheet.max_column).value:
+                try:
+                    nodegroup_id = worksheet.cell(2, worksheet.max_column).value
+                    graphid = str(Node.objects.filter(nodegroup_id=nodegroup_id)[0].graph_id)
+                    break
+                except: # IndexError, ValidationError
```

Comment:
Could you change this one too?
`except: (IndexError, ValidationError):`
I think you'll need to add an import for the validationerror: 
```from django.core.exceptions import ValidationError```

Reply:
same here.